src/main/java/wallet/command/Command.java

```diff
@@ -0,0 +1,156 @@
+                    System.out.println(count + "." + t.toString());
+                    count++;
+                }
+            } else if (command[1].equals("contact")) {
+                int count = 1;
+                System.out.println("Here are the contacts in your list:");
+                for (Contact c : contactList.getContactList()) {
+                    System.out.println(count + "." + c.toString());
+                    count++;
+                }
+            } else if (command[1].equals("expense")) {
+                int count = 1;
+                System.out.println("Here are the expenses in your list:");
+                for (Expense e : expenseList.getExpenseList()) {
+                    System.out.println(count + ". " + e.toString());
+                    count++;
+                }
+            }
+        } else if (command[0].equals("find")) {
+            int count = 1;
+            try {
+                String keyword = command[1];
+                System.out.println("Here are the matching tasks in your list:");
+                for (Task t : taskList.getTaskList()) {
+                    if (t.getDescription().contains(keyword)) {
+                        System.out.println(count + "." + t.toString());
+                        count++;
+                    }
+                }
+            } catch (ArrayIndexOutOfBoundsException e) {
+                System.out.println("☹ OOPS!!! Please specify a keyword to search for.");
+            }
+        } else if (command[0].equals("done")) {
+            try {
+                int num = Integer.parseInt(command[1]) - 1;
+                Task task = taskList.getTask(num);
+                if (task.getStatus()) {
+                    System.out.println("This task is already done.");
+                    System.out.println(task.toString());
+                } else {
+                    task.markAsDone();
+                    taskList.modifyTask(num, task);
+                    System.out.println("Nice! I've marked this task as done:");
+                    System.out.println(task.toString());
+                    //fileIO.updateToFile(task, num);
+                }
+            } catch (IndexOutOfBoundsException e) {
+                System.out.println("☹ OOPS!!! I'm sorry, but this task does not exist");
+            }
+        } else if (command[0].equals("todo") || command[0].equals("deadline")
+                || command[0].equals("event") || command[0].equals("dowithin")) {
+            try {
+                System.out.println("command[1]: " + command[1]);
+                Task task = taskList.createTask(command[0], command[1]);
+                if (task != null) {
+                    taskList.addTask(task);
+                    System.out.println("Got it. I've added this task:");
+                    System.out.println(task.toString());
+                    System.out.println("Now you have " + taskList.getTaskListSize() + " tasks in the list.");
```

Comment:
Avoid the use of magic string/numbers. Save these messages in a central location to be used by multiple classes/methods and etc. Would save trouble when writing tests as well.